packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -0,0 +1,331 @@
+<script lang="ts">
+import {
+  faArrowUpRightFromSquare,
+  faEllipsisVertical,
+  faPlay,
+  faRotateRight,
+  faStop,
+  faTrash,
+} from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { providerInfos } from '../../stores/providers';
+import type { ProviderContainerConnectionInfo, ProviderInfo } from '../../../../main/src/plugin/api/provider-info';
+import { onDestroy, onMount } from 'svelte';
+import type { IConfigurationPropertyRecordedSchema } from '../../../../main/src/plugin/configuration-registry';
+import { configurationProperties } from '../../stores/configurationProperties';
+import type { ContainerProviderConnection } from '@tmpwip/extension-api';
```

Comment:
FYI we rename it to `@podman-desktop/api` last week so you'll probably need to rebase the code